core/src/main/java/org/mskcc/cbio/oncokb/model/clinicalTrialsMathcing/Trial.java

```diff
@@ -10,10 +10,15 @@
 public class Trial {
     private String briefTitle;
     private String currentTrialStatus;
+    private String currentTrialStatusDate;
```

Comment:
Do you think it makes more sense to convert it to a Date or Instant? Will we use it as a Date or simply just return it?

Reply:
I don't believe this value is currently used or displayed anywhere but I had set it as a String because the format of the date in AACT was not consistent